onnx/backend/test/case/node/scatter.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np  # type: ignore
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+class Scatter(Base):
+
+    @staticmethod
+    def export_scatter_0():  # type: () -> None
```

Comment:
I would suggest using better names for these test points. For example, we can eliminate the explicit use of the `axis=0` attribute here, and call this `export_without_axis`. In the next test point, we can have the axis attribute (`axis=1`) and that can be called `export_with_axis`.

Reply:
fixed